jbmc/regression/jbmc/classpath-class-incorrect-package/test.desc

```diff
@@ -1,11 +1,9 @@
-KNOWNBUG
-Test.class
+CORE
```

Comment:
:no_entry_sign: This absolutely should be in its own commit

Reply:
Unfortunately all changes to the tests need to go into the same commit together with changing the interface (including the one that is currently still split out).